server/start/monitoring/index.ts

```diff
@@ -176,19 +177,16 @@ async function checkTemplate(context) {
  */
 async function insertMonitoringDataElasticsearch(context, data) {
   const monitoringIndexName = MONITORING_INDEX_PREFIX + indexDate(MONITORING_CREATION);
-  try {
     if (!MONITORING_ENABLED){
       return;
     };
-    try{
+    tryCatchForIndexPermissionError(monitoringIndexName, 'monitoring:insertMonitoringDataElasticsearch') (async() => {
```

Comment:
This  string`'monitoring:insertMonitoringDataElasticsearch'` could be on a `const` ?

Reply:
Amazing idea, thanks you!